### PR TITLE
Reformat typescript using prettier

### PR DESCRIPTION
**Pull request by ssbarnea**
That enables typescript on pre-commit prettier run as currently almost any attempt
to edit .ts files using vscode would introduce undesired reformatting.
That is because we recommend prettier extension which auto-reformats.

The only manually made change is the one line in pre-commit config, the rest is just pre-commit itself.

### CODE REVIEW

**Review by ganeshrn — APPROVED** [review]
Thanks!

**Review by cidrblock — APPROVED** [review]
I am admittedly not completely familiar with the code base, but this seems like a good idea as it will bring consistency to the repo without anyone having to lift a finger and smooth the path for future contributors.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -92,11 +92,11 @@ repos:
   - repo: https://github.com/igorshubovych/markdownlint-cli
     rev: v0.30.0
     hooks:
-    - id: markdownlint
-      exclude: >
-        (?x)^
-          docs/changelog-fragments\.d/\d+\.\w+\.md
-        $
+      - id: markdownlint
```
Comment (webknjaz) [inline .pre-commit-config.yaml:95]: This is an unrelated reformatting. Such changes must not be included.

File: test/utils/helper.ts [hunk 2]
```diff
@@ -2,14 +2,10 @@ import { TextDocument } from 'vscode-languageserver-textdocument';
 import * as path from 'path';
 import { promises as fs } from 'fs';
 
-
```
Comment (webknjaz) [inline test/utils/helper.ts:5]: Imports should have two line separator. This change is wrong.

File: test/utils/runCommand.test.ts [hunk 3]
```diff
@@ -4,19 +4,36 @@ import { WorkspaceManager } from '../../src/services/workspaceManager';
 import { createConnection } from 'vscode-languageserver/node';
 import { getDoc, isWindows } from './helper';
 
-
```
Comment (webknjaz) [inline test/utils/runCommand.test.ts:7]: Same problem here.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Pull request merged**